docs/concepts/package-paths.md

```diff
@@ -0,0 +1,109 @@
+---
+id: package-paths
+---
+
+# Package Paths
+
+A package path is a unique identifier for each package/realm. It specifies the 
+location of the package source code which helps differentiate it from others. 
+You can use a package path to:
+
+- Call a specific function from a package/realm. (e.g using `gnokey maketx call`)
+- Import it in other packages/realms.
+
+Here's a breakdown of the structure of a package path:
+
+- Domain: The domain of the blockchain where the package is deployed. 
+Currently, only `gno.land/` is supported.
+- Type: Defines the type of package.
+    - `p/`: [Package](packages.md)
+    - `r/`: [Realm](realms.md)
+- Namespace: A namespace can be included after the type (e.g., user or organization name). Namespaces are a
+way to group related packages or realms. A user can register a namespace for 
+themselves, or use the anonymous namespace. Read more [below](#namespaces).
+
+Here are some examples of package paths:
+
+- `gno.land/p/demo/avl`: This signifies a package named `avl` within the `demo` namespace.
+- `gno.land/r/gnoland/home`: This signifies a realm named `home` within the `gnoland` namespace.
+
+## Namespaces
+
+Namespaces provide users with the exclusive ability to publish code under
+their designated namespaces, similar to GitHub's user and organization model.
+A namespace follows these rules:
+  - Needs to be between 6 and 17 characters.
+  - Can only contain alphanumeric characters.
+  - No special characters are allowed (except underscore).
+  - Cannot consist solely of underscores. A namespace must have at least one 
+allowed alphanumeric character.
+  - Cannot start with a number. A namespace must begin with a letter.
+  - Cannot end with a trailing slash (`/`).
+
+:::warning Namespaces on gno.land testnets
+
+This feature is currently only enabled on the [Test4 testnet](./testnets.md#test4).
```

Comment:
Maybe reword this to clarify it is enabled on all networks from test4 and beyond, excluding portal loop. This way it won't have to be updated when we launch (test5? 😅 ) mainnet.